tensorflow/contrib/cmake/README.md

```diff
@@ -45,7 +45,7 @@ bindings.
 
 ### Pre-requisites
 
-* CMake version 3.5 up to 3.6
+* CMake version 3.5 up to 3.6 or 3.8
```

Comment:
Based on the latest information by @guschmue Let's go with  "Cmake version 3.6 or later"